src/types/challenges.types.ts

```diff
@@ -1,19 +1,35 @@
-export type challenges = {
-    id: number;
-    name: string,
+export type Hint = {
+    value: string,
+    cost: number
+};
+
+export type Challenges = {
+    challenge_id: string;
```

Comment:
looks good but we're missing the creator_name field here. i have an incoming PR so i can just add it in mine